topcoffea/modules/corrections.py

```diff
@@ -132,7 +133,7 @@ def AttachElectronSF(electrons, year=2018):
   # eta = np.abs(electrons.eta)
   eta = electrons.eta
   pt = electrons.pt
-
+  if '2016' in year: year=2016
```

Comment:
@andrea21z, the years should be strings, also I think it might be safer to check for an exact matche, e.g.:
```
if year == "2016APV": 
    year = "2016"
```